examples/Titanic/README.md

```diff
@@ -0,0 +1,63 @@
+#Titanic features attribution analysis using Captum and TorchServe.
+
+In this example, we show how to use a pre-trained custom Titanic model to performing real time classification prediction(survived/Not survived) and features attributions with Captum and TorchServe. You can download the dataset from https://biostat.app.vumc.org/wiki/pub/Main/DataSets/titanic3.csv 
+
+The inference service would return the prediction and attribution socre of features for a given test record.
+
+### Running the code
+
+To run the example via MLflow, navigate to the `examples/Titanic/` directory and run the command
+
+```
+mlflow run .
+
+```
+
+This will run `titanic_captum_interpret.py` with the default set of parameters such as `--max_epochs=100`. You can see the default value in the MLproject file.
+
+In order to run the file with custom parameters, run the command
+
+```
+mlflow run . -P max_epochs=X
+```
+
+where X is your desired value for max_epochs.
+
+If you have the required modules for the file and would like to skip the creation of a conda environment, add the argument --no-conda.
+
+```
+mlflow run . --no-conda
+```
+
+# Above commands will train the titanic model for further use.
+
+
+# Serve a custom model on TorchServe
+
+Run the commands given in following steps from the example dir. For example, if you cloned the repository into /home/my_path/serve, run the steps from /home/my_path/CaptumExample
```

Comment:
@Arvind-Ideas2IT Thanks for the PR,  would you please clarify on this, now the example is residing on "/home/ubuntu/mlflow-torchserve/examples/Titanic" should the commands be run from "/home/ubuntu/mlflow-torchserve/examples"?

Reply:
@HamidShojanazeri  Sorry, I have updated the README. The command should run from ./examples/Titanic dir.